### PR TITLE
Add isolarcloud.com #210

### PR DESCRIPTION
**Pull request by jslawler**

### CODE REVIEW

File: source/companies.json [hunk 1]
```diff
@@ -76,6 +76,11 @@
             "websiteUrl": "http://www.google.com",
             "description": "Google LLC is an American multinational technology company that specializes in Internet-related services and products, which include online advertising technologies, search engine, cloud computing, software, and hardware."
         },
+        "isolarcloud": {
```
Comment (jellizaveta) [inline source/companies.json:79]: Maybe `Sungrow Power Supply Company Limited` could be added to the company and as a product `isolarcloud` to the trackers

**Review by jellizaveta — CHANGES_REQUESTED** [review]

File: source/companies.json [hunk 2]
```diff
@@ -76,6 +76,11 @@
             "websiteUrl": "http://www.google.com",
             "description": "Google LLC is an American multinational technology company that specializes in Internet-related services and products, which include online advertising technologies, search engine, cloud computing, software, and hardware."
         },
+        "sungrow": {
+            "name": "Sungrow Power Supply Company Limited",
+            "websiteUrl": "https://sungrowpower.com/",
+            "description": "Sungrow is the world’s most bankable inverter brand with over 340 GW installed worldwide as of December 2022. Founded in 1997 by University Professor Cao Renxian, Sungrow is a leader in the research and development of solar inverters with the largest dedicated R&D team in the industry and a broad product portfolio offering PV inverter solutions and energy storage systems for utility-scale, commercial & industrial, and residential applications, as well as internationally recognized floating PV plant solutions, NEV driving solutions, EV charging solutions and renewable hydrogen production systems. With a strong 26-year track record in the PV space, Sungrow products power over 150 countries worldwide. "
```
Comment (Alex-302) [inline source/companies.json:82]: Is not it too long? Description is not a biography of a company:)

**Review by jellizaveta — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Comment by jellizaveta** [comment]
@jslawler Could you pull the current master in your branch, because changes were made in the repository and workflows modified.

**Comment by jellizaveta** [comment]
Can I ask you to do the same for the other prs to pass the checks properly?

**Pull request merged**